plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -393,15 +393,14 @@ class ServicesTable extends React.Component {
   }
 
   renderInstances(prop, service) {
+    const isFolder = service.isFolder();
     const instancesCount = service.getInstancesCount();
     const tasksRunning = service.getTaskCount();
     const overview = tasksRunning === instancesCount
       ? ` ${tasksRunning}`
       : ` ${tasksRunning}/${instancesCount}`;
 
-    const content = instancesCount == null || service.list
-      ? "\u2014"
-      : overview;
+    const content = instancesCount == null || isFolder ? "\u2014" : overview;
```

Comment:
Wonder why you _cache_ the result of `isFolder` in line 396 instead of calling the getter right in this expression?